### PR TITLE
and env vars and mongo config for filter-flex-api

### PR DESCRIPTION
**Pull request by ChrysmOre**
## What
- Update environment variables and config for dp-cantabular-filter-flex-api

# How
Check changes make sense

# Who
Anyone

### CODE REVIEW

File: cantabular-import/dp-cantabular-filter-flex-api.yml [hunk 1]
```diff
@@ -15,9 +15,15 @@ services:
         ports:
             - 27100:27100
         restart: unless-stopped
+        depends_on:
+            - kafka-1
+            - kafka-2
```
Comment (redhug1) [inline cantabular-import/dp-cantabular-filter-flex-api.yml:20]: Is this line needed, please ask David Subiros

File: cantabular-import/dp-cantabular-filter-flex-api.yml [hunk 2]
```diff
@@ -15,9 +15,15 @@ services:
...
+            - kafka-3
```
Comment (redhug1) [inline cantabular-import/dp-cantabular-filter-flex-api.yml:21]: Is this line needed, please ask David Subiros

File: cantabular-import/dp-cantabular-filter-flex-api.yml [hunk 3]
```diff
@@ -15,9 +15,15 @@ services:
...
+            - mongodb
         environment:
             BIND_ADDR:                   ":27100"
-            SERVICE_AUTH_TOKEN:          $SERVICE_AUTH_TOKEN
+            SERVICE_AUTH_TOKEN:           $SERVICE_AUTH_TOKEN
```
Comment (redhug1) [inline cantabular-import/dp-cantabular-filter-flex-api.yml:25]: Should this line be:
            SERVICE_AUTH_TOKEN:                 "testToken"

, please ask David Subiros

File: cantabular-import/dp-cantabular-filter-flex-api.yml [hunk 4]
```diff
@@ -15,9 +15,15 @@ services:
...
+            KAFKA_ADDR:                  "kafka-1:19092,kafka-2:19092,kafka-3:19092"
```
Comment (redhug1) [inline cantabular-import/dp-cantabular-filter-flex-api.yml:26]: Should this line be:
            KAFKA_ADDR:                         "kafka-1:9092"

, please ask David Subiros

**Review by redhug1 — COMMENTED** [review]
Please check my questions with David Subiros.
Thanks.

File: cantabular-import/dp-cantabular-filter-flex-api.yml (hunk 4 above)
Comment (ChrysmOre) [reply cantabular-import/dp-cantabular-filter-flex-api.yml:26]: I think you're getting confused with the component test yml files. The answer is 'no' to the above questions... check the other yml files in this directory you'll see they're set up the same.

**Review by redhug1 — APPROVED** [review]
LGTM

**Pull request merged**